assets/src/edit-story/components/panels/link.js

```diff
@@ -53,10 +53,14 @@ const Note = styled.span`
 `;
 
 function LinkPanel({ selectedElements, onSetProperties }) {
+  const y = getCommonValue(selectedElements, 'y');
   const link = getCommonValue(selectedElements, 'link') || null;
   const isFill = getCommonValue(selectedElements, 'isFill');
+  const inferredLinkType = inferLinkType(y);
```

Comment:
Just to confirm: is this solely for the "new link" case? And in either case, let's use `useMemo()` here - that math will be getting more nuanced.